lib/project/web-site-project.ts

```diff
@@ -0,0 +1,112 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import path = require("path");
+import util = require("util");
+
+import frameworkProjectBaseLib = require("./framework-project-base");
+import helpers = require("../common/helpers");
+
+export class MobileWebSiteProject extends frameworkProjectBaseLib.FrameworkProjectBase implements Project.IFrameworkProject {
+	constructor(private $config: IConfiguration,
+		$fs: IFileSystem,
+		$logger: ILogger,
+		private $projectConstants: Project.IProjectConstants,
+		private $projectFilesManager: Project.IProjectFilesManager,
+		private $templatesService: ITemplatesService,
+		$resources: IResourceLoader) {
+		super($logger, $fs, $resources);
+	}
+
+	public get name(): string {
+		return this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.MobileWebSite;
+	}
+
+	public get capabilities(): IProjectCapabilities {
+		return {
+			build: false,
+			buildCompanion: false,
+			deploy: false,
+			simulate: true,
+			livesync: false,
+			livesyncCompanion: false,
+			updateKendo: false
+		};
+	}
+
+	public get defaultProjectTemplate(): string {
+		return this.$config.DEFAULT_WEBSITE_PROJECT_TEMPLATE;
+	}
+
+	public get liveSyncUrl(): string {
+		return "";
+	}
+
+	public get requiredAndroidApiLevel(): number {
+		return 10;  // 2.3 Gingerbread
+	}
+
+	public get configFiles():  Project.IConfigurationFile[] {
+		return [];
+	}
+
+	public getProjectTargets(projectDir: string): IFuture<string[]> {
+		var result: string[] = [];
+		return (() => { return result; }).future<string[]>()();
+	}
+
+	public getTemplateFilename(name: string): string {
+		return util.format("Telerik.Mobile.MobileWebSite.%s.zip", name);
+	}
+
+	public alterPropertiesForNewProject(properties: any, projectName: string): void {
+		this.alterPropertiesForNewProjectBase(properties, projectName);
+	}
+
+	public projectTemplatesString(): IFuture<string> {
+		return this.$templatesService.getTemplatesString(/.*Telerik\.Mobile\.MobileWebsite\.(.+)\.zip/);
+	}
+
+	public getProjectFileSchema(): IFuture<any> {
+		return (() => { }).future<any>()();
+	}
```

Comment:
Why this function is not implemented? It should return the mobile web site schema, like the other classes do.